lib/taskana-core/src/main/java/pro/taskana/task/internal/ServiceLevelHandler.java

```diff
@@ -303,7 +303,8 @@ private boolean plannedHasChanged(Task newTask, Task oldTask) {
 
   private Instant calculateDue(Instant planned, Duration duration) {
     Instant dueExclusive = workingTimeCalculator.addWorkingTime(planned, duration);
-    if (!planned.equals(dueExclusive)) {
+    if (taskanaEngine.getEngine().getConfiguration().isUseWorkingTimeCalculation()
+        && !planned.equals(dueExclusive)) {
       // Calculation is exclusive, but we want due date to be inclusive. Hence, we subtract a
       // millisecond
```

Comment:
Is this comment correct? There is no subtraction in this if part of the if statement...

Reply:
Oh nooo 🤦  - sorry 😄